src/components/IdentityLine.tsx

```diff
@@ -2,11 +2,7 @@ import Link from "next/link";
 import React from "react";
 import styled from "styled-components";
 
-import {
-  IdentityTypes,
-  Identity,
-  ReceivedIdentityTypes,
-} from "../@types/Identity";
+import { Identity } from "../@types/Identity";
```

Comment:
I just discovered this new TS feature: [import type](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html).

It adds more readability on imports, and also a couple of goodies. If you could do it on the files you changed here.

Be careful not to use it on `enums` thought (because enums are transpiled to objects)!

```suggestion
import type { Identity } from "../@types/Identity";
```

Reply:
OK fixed 6994e62 I could not use this syntax in files where a use a type like a value though.
For example in this kind of implementation:
`
        <p>
          {type === ReceivedIdentityTypes.PHONE
            ? "phone number"
            : "email address"}{" "}
          *
        </p>
`

Reply:
> I could not use this syntax in files where a use a type like a value though.
Yes, these are called enum, that's what I was talking about in my previous message :).